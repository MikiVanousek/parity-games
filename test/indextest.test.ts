import * as fs from 'fs';
import * as path from 'path';

it.skip('should match the snapshot of index.html', () => {
    const filePath = path.resolve(__dirname, '../dist/index.html');
    const fileContent = fs.readFileSync(filePath, 'utf-8');
    expect(fileContent).toMatchSnapshot();
});