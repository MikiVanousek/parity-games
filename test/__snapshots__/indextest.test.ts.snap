// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot of index.html 1`] = `
"<!DOCTYPE html>
<html lang="en">


<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document Title</title>

</head>

<body>
    <button id="test-btn">Test!</button>
    <script src="main.js"></script>
</body>

</html>"
`;
